src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mans = people.filter(human => {
+    return human.sex !== 'm'
+      ? false : century === undefined
+        ? true : Math.ceil(human.died / 100) === century;
+  });
```

Comment:
avoid if inside if

You can choose the right callback dependence on the century